src/post/CompactPostThread.tsx

```diff
@@ -104,7 +104,7 @@ const PreviousContent = (props: CompactThreadProps) => {
                 totalComments={post.totalComments}
                 newContributions={post.newContributions}
                 newComments={post.newComments}
-                notesLink={post.notesLink}
+                notesLink={post.createdTimeLink}
```

Comment:
I'm open for suggestions. It's possible that we just need to add more linking options for CompactPost, since it's possible that consumers will want the intermediate "notes" to have a different interaction than the ones at the end (right now they aren't even that visually distinctive, but they should be more so later).

I think the issue with naming them based on what they do is that I assume it's legit to make different choices there in terms of views/clients.